### DISCUSSION TITLE
CS50 - 转码路线图

### DISCUSSION THREAD

**Discussion by giscus** [wangzhe3224/zhuan-ma#9] [Q&A] [answered]
# CS50 - 转码路线图

转码路线图

https://wangzhe3224.github.io/zhuan-ma/core/cs50/

**Comment by Rainohhh** [comment, marked as answer]
点击链接进去白屏

**Reply by Rainohhh** [reply]
链接进不去呀

**Reply by Lexi-one** [reply]
我也点不开

**Reply by wangzhe3224** [reply]
难道被墙了？？我这边可以看呀

**Reply by wangzhe3224** [reply]
放一个 B 站链接： https://www.bilibili.com/video/BV1nh411J7Wq/?vd_source=da65fd0f11a3d90e543b48cf26e65fb7

**Reply by Jingsi1108** [reply]
bilibili的视频有5个课时，是CS50所有的视频不？

**Comment by skylong12** [comment]
确实是白屏，无法加载出内容，等梯子好了再试试
